src/engraving/dom/anchors.cpp

```diff
@@ -100,11 +99,25 @@ TimeTickAnchor* EditTimeTickAnchors::createTimeTickAnchor(Measure* measure, Frac
         anchor->setParent(segment);
         anchor->setTrack(track);
         segment->add(anchor);
+        if (System* system = measure->system()) {
+            if (Page* page = system->page()) {
+                page->invalidateBspTree();
```

Comment:
Please explain why you need it here. It is quite confusing to see it in a create function...